private/dir.go

```diff
@@ -0,0 +1,55 @@
+package private
```

Comment:
I'm not sure I explained myself well but `private.ResolvePathToFullyExplicit` doesn't tell me much about `private`